bin/commonmark

```diff
@@ -77,16 +75,12 @@ if (format === 'html') {
 }
 
 if (files.length === 0) {
-   if (os.platform() === "win32") {
-     inps.push(fs.readFileSync(0, 'utf-8'));
-   } else {
-     inps.push(fs.readFileSync('/dev/tty', 'utf-8'));
-   }
-} else {
-  for (i = 0; i < files.length; i++) {
-    var file = files[i];
-    inps.push(fs.readFileSync(file, 'utf8'));
-  }
+  files = ['/dev/stdin'];
+}
+
+for (i = 0; i < files.length; i++) {
+  var file = files[i];
+  inps.push(fs.readFileSync(file, 'utf8'));
```

Comment:
Was there a reason for removing the platform-specific handling here?
cf. discussion in #198.

Reply:
No reason for removing the platform stuff in the bin script. Looks like it was done in the same commit as ESM-ification: 9bcf45b1e384eb0983d023298e0e65bffafceb76. So `git revert` rolled those changes back, too.

I can replace them, or you can fee free to. Whatever's easiest for you.